runtime/src/bank.rs

```diff
@@ -900,16 +900,38 @@ impl Bank {
             &genesis_config.rent,
         );
 
+        // Add default static programs
+        self.add_static_program(
+            "system_program",
+            solana_sdk::system_program::id(),
+            system_instruction_processor::process_instruction,
+        );
+        self.add_static_program(
+            "config_program",
+            solana_config_program::id(),
+            solana_config_program::config_processor::process_instruction,
+        );
+        self.add_static_program(
+            "stake_program",
+            solana_stake_program::id(),
+            solana_stake_program::stake_instruction::process_instruction,
+        );
+        self.add_static_program(
+            "vote_program",
+            solana_vote_program::id(),
+            solana_vote_program::vote_instruction::process_instruction,
+        );
+
         // Add additional native programs specified in the genesis config
         for (name, program_id) in &genesis_config.native_instruction_processors {
```

Comment:
The genesis_config for mainnet-beta, testnet, and devnet all have the now static programs in `native_instruction_processors`, so I think if one tried to start from genesis again then we'd get a conflict here.   This generally shouldn't matter but I think we could probably avoid the conflict if `add_native_program` did nothing should an account at `program_id` already exist

Reply:
Ok, I removed that assert